src/uct/ib/rc/accel/rc_mlx5.inl

```diff
@@ -92,10 +103,13 @@ uct_rc_mlx5_iface_hold_srq_desc(uct_rc_mlx5_iface_common_t *iface,
         uct_recv_desc(udesc) = release_desc;
         seg->srq.ptr_mask   &= ~UCS_BIT(stride_idx);
     } else {
-        udesc                = UCS_PTR_BYTE_OFFSET(seg->srq.desc, offset);
-        uct_recv_desc(udesc) = release_desc;
-        seg->srq.ptr_mask   &= ~1;
-        seg->srq.desc        = NULL;
+        seg->srq.ptr_mask &= ~UCS_BIT(UCT_IB_RX_SG_PAYLOAD_IDX);
+        if (iface->super.super.super.rx_allocator.release_payload_desc) {
```

Comment:
rename iface->super.super.super.rx_allocator.release_payload_desc to "user_allocator" - but inverse the meaning